templates/helpers.js

```diff
@@ -2,6 +2,12 @@ var fs = require("fs");
 var path = require("path");
 var escapeHTML = require("escape-html");
 var unescapeHTML = require("unescape-html");
+var generateTitle = require("./generate-title");
```

Comment:
When I `npm run styles`, I get the error: `Error: Cannot find module './generate-title'`

Reply:
My bad I wrote a shared code in a file and I removed it!

Reply:
Is fixed now